src/Shared/BuildEnvironmentHelper.cs

```diff
@@ -383,6 +383,11 @@ private static bool IsProcessInList(string processName, string[] processList)
         private static string GetProcessFromRunningProcess()
         {
 #if RUNTIME_TYPE_NETCORE
+            if (AssemblyUtilities.EntryAssembly == null)
```

Comment:
Also, either add a test for this scenario or a comment on how the AssemblyUtilities.EntryAssembly can be null justifying WHY this conditional is needed.